src/templates/ml-facets-inline.html

```diff
@@ -4,8 +4,10 @@
     <div ng-repeat="value in facet.facetValues" ng-class="{ 'on': value.selected} ">
       <a ng-click="toggle({facet: facet.__key, value: value.name})" title="{{ value.name }}">
         <i ng-class="{ 'fa fa-trash-o': value.selected }"></i>
-        {{ value.name | truncate:20 }} ({{ value.count }})
+        <span ng-if="value.name">{{ value.name | truncate:truncateLength }}</span>
```

Comment:
How about using the built-in limitTo filter? I do see use for a custom truncate if it would tokenize, and truncate whole words only or something, but it doesn't see to do that (yet)..